lib/ViewAllLogsFilters/ViewAllLogsFilters.js

```diff
@@ -92,8 +92,6 @@ export const ViewAllLogsFilters = ({
     // sets current tenant's timezone
     moment.tz.setDefault(timezone);
     onChange(filterData);
-    // restore to a default timezone
-    moment.tz.setDefault();
```

Comment:
any reason to do this? this will affect all places where moment is used

Reply:
@NikitaSedyx i made an clean up